data_steward/cdr_cleaner/manual_cleaning_rules/remediate_basics.py

```diff
@@ -146,12 +151,16 @@
 FROM `{{project}}.{{incremental_dataset}}.{{table}}` i
 JOIN `{{project}}.{{obs_id_lookup_dataset}}.{{new_obs_id_lookup}}` oim
 ON i.observation_id = oim.source_observation_id
+{% if exclude_lookup_dataset and exclude_lookup_table %}
 WHERE i.observation_id IN (
```

Comment:
@hiro-mishima - Do we need the WHERE clause on 155-156?  The inner WHERE clause on 157-160 should suffice if not AIAN, unless I'm missing something (which is very possible)?

Reply:
@molly-giddings We need these two WHERE clauses because this JINJA template is for observation mapping table. Observation mapping table does not have `person_id` column. So we need to reference observation table for `person_id`.